_posts/2016-02-10-solita-developer-manifesto.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Solita Developer Manifesto
+author: Kauko
+excerpt: An attempt at putting to words what makes Solita a great place to work, and what is expected of developers, new and old alike.
+tags:
+- Culture
+- Solita
+---
+
+The Agile Manifesto, first published in 2001, is a document that is familiar to many. Aiming to “uncover better ways to develop software”, it defines a set of principles written in the form of “X over Y”, meaning that “while there is value in the items on the right, we value the items on the left more.” This way of presenting the thoughts of the manifesto is not only concise yet expressive, but also helps drive home the point that the manifesto is about prioritization, not about elimination. 
+
+At Solita we place a very large emphasis on the importance of workplace culture. Great culture stems from recruiting the right kinds of people, and then encouraging those people, and giving them the tools, to keep making Solita one of the best places to work in Finland. Perhaps in accordance with the Agile Manifesto, Solita does not have a set of dusty corporate documents defining the required code of conduct, but rather we refer to our four **Core Values** *(passion, boldness, casualness, caring)* that were identified by the employees themselves. Maybe due to the growth that Solita has been blessed to experience in the last few years, a handful of more senior developers wanted to create a document to complement the Core Values. This document is called the *Solita Developer Manifesto*, and, as the name would suggest, its audience are us, the developers.
+
+Reviewing the Principles
+---
+
+The Solita Developer Manifesto is comprised of four principles. With the help of these principles, the manifesto wants to help developers, juniors and seniors alike, understand and agree what is expected of them. Originally they were written in Finnish, so naturally some of the nuances may have been lost in translation.
```

Comment:
agree -> agree on, prepositio
manifesto wants - luulisin että manifesto aims olisi korrektimpi, koska manifesto ei ole henkilö.
originally they  -> originally it. Edellinen lause puhuu yksikössä manifestosta.